### ISSUE TITLE
less@3.10.1 bug (actually less-loader bug)

### ISSUE THREAD

**Issue by kwangkim** [apollographql/gatsby-theme-apollo#53]
`less@3.10.1` may have a bug.

Ref: https://github.com/less/less.js/issues/3414
I have a following error. 
```
ERROR in ./node_modules/gatsby-theme-apollo/src/styles.less (./node_modules/css-loader??ref--16-oneOf-1-1!./node_modules/postcss-loader/lib??postcss-3!./node_modules/less-loader/dist/cjs.js??ref--16-oneOf-1-3!./node_modules/gatsby-theme-apollo/src/styles.less)
Module build failed (from ./node_modules/less-loader/dist/cjs.js):


Class constructor FileManager cannot be invoked without 'new'
      in undefined (line undefined, column undefined)
 @ ./node_modules/gatsby-theme-apollo/src/styles.less 2:14-179 21:1-42:3 22:19-184
 @ ./node_modules/gatsby-theme-apollo/src/components/layout.js
 @ ./node_modules/gatsby-theme-apollo/index.js
 @ ./src/components/template.js
 @ ./.cache/sync-requires.js
 @ ./.cache/app.js
 @ multi ./node_modules/event-source-polyfill/src/eventsource.js (webpack)-hot-middleware/client.js?path=/__webpack_hmr&reload=true&overlay=false ./.cache/app
```
To avoid this error, I did downgrade less to `less@3.9.0` by adding
```
  "resolutions": {
    "less": "3.9.0"
  },
```
in my package.json.

**Comment by kwangkim** [comment]
Or you may upgrade `less-loader@4.1.0` into `less-loader@5.0.0`.

https://github.com/less/less.js/issues/3414#issuecomment-522283588

**Comment by trevorblades** [comment]
Fixed by #55! You can install `gatsby-theme-apollo@latest` or `gatsby-theme-apollo-docs@latest` to get the fix.
